src/main/java/com/agilethought/atsceapi/validator/user/UserValidator.java

```diff
@@ -38,7 +42,13 @@ public void validate(User user) {
         validateType(user.getType());
         validateFirstName(user.getFirstName());
         validateLastName(user.getLastName());
-        validateEmail(user.getEmail());
+        validateEmailFormat(user.getEmail());
+        // If there is no value in the user's id,
+        // it's a new user being created.
+        if (user.getId() == null)
```

Comment:
what happen if user send and empty string in his request? like `user_id=""`

Reply:
When the method
`public UpdateUserResponse updateUser(UpdateUserRequest request, String id) {`
from UserService is called, it first verifies that a user can be found by the provided id (if any). When the id is an empty String, the API will return a 405 response with the message "Request method 'PUT' not supported" before getting into the execution of methods inside UserValidator class. However, if it's a better approach, I could use the
`public static boolean isValidString(String str) {`
from ValidationUtils class for this validation.